spec/lib/audit/catalog_to_moab_class_spec.rb

```diff
@@ -6,31 +6,41 @@
   let(:storage_dir) { 'spec/fixtures/storage_root01/moab_storage_trunk' }
   let(:limit) { Settings.c2m_sql_limit }
 
-  context '.check_version_on_dir_of_batch' do
+  context '.check_version_on_dir' do
     include_context 'fixture moabs in db'
-    let(:subject) { described_class.check_version_on_dir_of_batch(last_checked_version_b4_date, storage_dir, limit) }
+    let(:subject) { described_class.check_version_on_dir(last_checked_version_b4_date, storage_dir, limit) }
 
-    it 'creates an instance and calls #check_catalog_version' do
-      c2m_mock = instance_double(described_class)
-      allow(described_class).to receive(:new).and_return(c2m_mock)
-      expect(c2m_mock).to receive(:check_catalog_version).exactly(3).times
-      described_class.check_version_on_dir_of_batch(last_checked_version_b4_date, storage_dir, limit)
-    end
-  end
+    context 'when there are PreservedCopies to check' do
+      let(:c2m_mock) { instance_double(described_class) }
 
-  context ".check_version_on_dir" do
-    include_context 'fixture moabs in db'
-    let(:subject) { described_class.check_version_on_dir(last_checked_version_b4_date, storage_dir) }
+      it 'creates an instance and calls #check_catalog_version within the limit' do
```

Comment:
"within the limit" doesn't make sense to me.  ?? `when query results are in single batch, calls #check_catalog_version for every result` ???   Is that the gist??

Reply:
i think so

Reply:
Sounds good to me